internal/pkg/cli/deploy/workload.go

```diff
@@ -160,7 +162,7 @@ type WorkloadDeployerInput struct {
 	Name             string
 	App              *config.Application
 	Env              *config.Environment
-	ImageTag         string
+	RuntimeImage     RuntimeImage
```

Comment:
What do you think of renaming the struct and field like this?
```suggestion
	Image     ContainerImageIdentifier
```